src/lib/driver/message_handler.ts

```diff
@@ -30,6 +30,13 @@ export class DriverMessageHandler {
         return { config: dumpLogConfig(driver, client.schemaVersion) };
       case DriverCommand.updateLogConfig:
         driver.updateLogConfig(message.config);
+        clientsController.clients.forEach((cl) =>
+          cl.sendEvent({
```

Comment:
Should we filter out the current client?

```suggestion
          if(cl !== client) cl.sendEvent({
```

Reply:
I explicitly chose not to but I suppose if the events are coming in, the client is safe to update its internal state when it detects an update request